RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 12.3
 -----
- 
+ * Improved connectivity errors messaging in sharing screen.
```

Comment:
This update is a bit generic as there was more work on sharing-screen error messaging improvement.